components/ee/agent-smith/cmd/testbed/main.go

```diff
@@ -60,10 +60,7 @@ func run(args []string) error {
 				env[i] += e + ":" + cwd
 			}
 		}
-		env = append(env, "GITPOD_OWNER_ID=owner-id")
```

Comment:
i guess it was like that before for readability. I don't think inlining it makes more readable.

Reply:
From my point of view, this makes it more readable _(especially here)_ and at the same time allows a _(very)_ small performance improvement 🤔

Reply:
Sorry, I agree with @akosyakov here. I don't think this makes the code easier or better readable.

Reply:
As you wish, especially since the PR is more intended to evoke the possible _performance issues_ compared to the addition of multiple `append`. But I understand that in terms of readability it's less affordable.

Reply:
Thank you for your feedback, I didn't know that. So I close this PR 👍